broker/src/tunneldigger_broker/eventloop.py

```diff
@@ -57,6 +57,8 @@ def start(self):
 
                     if event & select.EPOLLIN:
                         pollable.read(file_object)
+                    elif event & select.EPOLLERR:
```

Comment:
You might want to handle `EPOLLHUP` as well here, if you are adding this: https://stackoverflow.com/questions/24119072/how-to-deal-with-epollerr-and-epollhup

`EPOLLHUP` is when the connection is closed nicely. It [looks like that those two events are always listened for](https://man7.org/linux/man-pages/man2/epoll_ctl.2.html).

Reply:
Good catch! I made it also close the pollable on HUP.